airflow/cli/commands/info_command.py

```diff
@@ -0,0 +1,427 @@
+                System PATH: [{system_path}]
+                Python PATH: [{python_path}]
+                airflow on PATH: [{airflow_on_path}]
+                """
+            )
+            .strip()
+            .format(
+                airflow_home=self.airflow_home,
+                system_path=os.pathsep.join(self.system_path),
+                python_path=os.pathsep.join(self.python_path),
+                airflow_on_path=self.airflow_on_path,
+            )
+        )
+
+
+class ConfigInfo:
+    """"Most critical config properties"""
+
+    def __init__(self, anonymizer: Anonymizer):
+        self.executor = configuration.conf.get("core", "executor")
+        self.dags_folder = anonymizer.process_path(
+            configuration.conf.get("core", "dags_folder", fallback="NOT AVAILABLE")
+        )
+        self.plugins_folder = anonymizer.process_path(
+            configuration.conf.get("core", "plugins_folder", fallback="NOT AVAILABLE")
+        )
+        self.base_log_folder = anonymizer.process_path(
+            configuration.conf.get("logging", "base_log_folder", fallback="NOT AVAILABLE")
+        )
+        self.base_log_folder = anonymizer.process_path(
+            configuration.conf.get("logging", "base_log_folder", fallback="NOT AVAILABLE")
+        )
+        self.sql_alchemy_conn = anonymizer.process_url(
+            configuration.conf.get("core", "SQL_ALCHEMY_CONN", fallback="NOT AVAILABLE")
+        )
+
+    def __str__(self):
+        return (
+            textwrap.dedent(
+                """\
+                Executor: [{executor}]
+                SQL Alchemy Conn: [{sql_alchemy_conn}]
+                DAGS Folder: [{dags_folder}]
+                Plugins Folder: [{plugins_folder}]
+                Base Log Folder: [{base_log_folder}]
+                """
+            )
+            .strip()
+            .format(
+                executor=self.executor,
+                sql_alchemy_conn=self.sql_alchemy_conn,
+                dags_folder=self.dags_folder,
+                plugins_folder=self.plugins_folder,
+                base_log_folder=self.base_log_folder,
+            )
+        )
+
+
+class ToolsInfo:
+    """The versions of the tools that Airflow uses"""
+
+    def __init__(self, anonymize: Anonymizer):
+        del anonymize  # Nothing to anonymize here.
+        self.git_version = self._get_version(["git", "--version"])
+        self.ssh_version = self._get_version(["ssh", "-V"])
+        self.kubectl_version = self._get_version(["kubectl", "version", "--short=True", "--client=True"])
+        self.gcloud_version = self._get_version(["gcloud", "version"], grep=b"Google Cloud SDK")
+        self.cloud_sql_proxy_version = self._get_version(["cloud_sql_proxy", "--version"])
+        self.mysql_version = self._get_version(["mysql", "--version"])
+        self.sqlite3_version = self._get_version(["sqlite3", "--version"])
+        self.psql_version = self._get_version(["psql", "--version"])
```

Comment:
Why did we pick psql, and not the version of psycopg2, say?

(Airflow never uses `psql`)

Reply:
Airflow uses psql here: https://github.com/apache/airflow/blob/master/airflow/cli/commands/db_command.py#L91

I did not check any version of the Python library. For this we have a very other command  - `pip freeze`. In the future, we can add selected libraries if necessary.